src/components/language-selector/language-selector.css

```diff
@@ -2,10 +2,20 @@
 @import "../../css/units.css";
 
 .group {
-    display: inline-flex;
-    flex-direction: row; /* makes columns, for each label/form group */
-    align-items: center;
-    vertical-align: middle;
+    z-index: 50;
+    top: $menu-bar-height;
+    left: 0;
+    position: absolute;
+    border: 1px solid $ui-black-transparent;
+    border-radius: 0 0 8px 8px;
+    background-color: $motion-primary;
+    padding: 0;
+    margin: 0;
+    min-width: 186px;
+    max-width: 260px;
```

Comment:
Such specific fixed values are pretty suspicious in CSS. It seems the same thing can be achieved by not setting a min/max width here and removing the width from `.language-select` below?